alacritty.yml

```diff
@@ -221,15 +221,15 @@ selection:
 
 dynamic_title: true
 
-hide_cursor_when_typing: false
-
-# Style of the cursor
-#
-# Values for 'cursor_style':
-# - Block
-# - Underline
-# - Beam
-cursor_style: Block
+cursor:
+  # Style of the cursor
+  #
+  # Values for 'cursor_style':
```

Comment:
This comment is slightly wrong `s/cursor_style/style` but I would compress the whole thing down to a single line (before the enumeration):

> `style` can be one of:
> ...

Reply:
I've reworked the documentation in the config, please let me know what you think about it. :)